### ISSUE TITLE
Whisper and low pitch don't apply to child voice

### ISSUE THREAD

**Issue by stacsk** [k2-fsa/OmniVoice#124]
### Checks

- [x] This template is only for usage issues encountered.
- [x] I have thoroughly reviewed the project documentation but couldn't find information to solve my problem.
- [x] I have searched for existing issues, including closed ones, and couldn't find a solution.
- [x] I am using English to submit this issue to facilitate community communication.

### Environment Details

python 3.10, torch 2.8.0

### Steps to Reproduce

1. Go to voice design
2. select gender
3. select child
4. select whisper or low pitch
5. fix the seed
6. generate voice, listen, see that it's adult voice
7. reproduce the steps without selecting "low pitch" or "whisper"
8. see that result is almost the same

### ✔️ Expected Behavior

low pitch or whisper settings should apply to child in voice design.

### ❌ Actual Behavior

result is adult voice, instead of child voice.

p.s.: I don't remember your gradio interface exactly since I switched to ComfyUI today. The instructions are for gradio interface, as well as I can remember it.

**Comment by snakefood3232** [comment]
I understand the issue with the whisper and low pitch settings not applying to child voices and resulting in adult voices. I'd handle this by reviewing the voice modulation logic in the codebase to ensure the settings are correctly applied to child voice profiles. I can have a PR up in 2 days to address this. I've previously tackled similar voice synthesis challenges in Python environments with success.

**Comment by zhu-han** [comment]
Some combinations of instructions may not work well due to limitations in the training data. For example, our training data for "whisper" speech does not have the age "child".
